### PR TITLE
From 8080 to 6502

### PR DESCRIPTION
**Pull request by gc-av**
- Changed instructions from 8080 to 6502
- Added documentation regarding 6502 register architecture
- Created bit setter/unsetter and getter functions

### CODE REVIEW

File: architecture.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# 6502 Architecture
+
+[http://archive.6502.org/books/mcs6500_family_hardware_manual.pdf](6502 Documentation)
```
Comment (intarga) [inline architecture.md]: You're missing the last link I sent btw

File: main.c [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+	uint8_t A;
+	uint8_t Y;
+	uint8_t X;
+	uint8_t PCH;
+	uint8_t PCL;
+	uint8_t S;
+	uint8_t P;
+} Cpu_state;
+
+// Set bit (0-7) in registry
```
Comment (intarga) [inline main.c:15]: Appreciate these docstrings :+1:

File: architecture.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+[https://www.westerndesigncenter.com/wdc/documentation/w65c02s.pdf](65C02 Documentation) - 65C02 is the static version of the 6502. This manual can be used, as long as the static features are not implemented
```
Comment (intarga) [inline architecture.md]: Also, your links are broken, check the markdown syntax

**Review by intarga — CHANGES_REQUESTED** [review]

**Review by intarga — APPROVED** [review]
LGTM, ship it!

**Review by odeta939 — APPROVED** [review]

**Pull request merged**